packages/image_picker/tizen/src/image_picker_tizen_plugin.cc

```diff
@@ -151,131 +140,155 @@ class ImagePickerTizenPlugin : public flutter::Plugin {
     } else {
       switch (permission) {
         case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ALLOW:
-          LOG_INFO("ppm_check_permission success! [%d]", (int)permission);
-          SetContentMimeType(mimeType);
+          LOG_INFO("ppm_check_permission success!");
           PickContent();
           return;
         case PRIVACY_PRIVILEGE_MANAGER_CHECK_RESULT_ASK:
-          ret = ppm_request_permission(privilege, AppRequestResponseCb, this);
+          ret = ppm_request_permission(
+              privilege,
+              [](ppm_call_cause_e cause, ppm_request_result_e result,
+                 const char *privilege, void *data) -> void {
+                auto *plugin = (ImagePickerTizenPlugin *)data;
```

Comment:
It was intentional because I didn't want to make too many style fixes in this PR. There are so many places where `static_cast` is not being used in this file.

Reply:
Oh... I understand. :)